flexget/plugins/notifiers/email.py

```diff
@@ -104,56 +101,47 @@ class EmailNotifier(object):
         'additionalProperties': False,
     }
 
-    def notify(self, to, message, title, smtp_host, smtp_port, smtp_username=None, smtp_password=None, smtp_tls=None,
-               smtp_ssl=None, html=None, **kwargs):
+    def notify(self, title, message, config):
```

Comment:
@liiight What you had was different, it was a single dict called "data" and contained message and configuration in same dict. This is what I proposed as first ideal solution but there were some challenges with that at the time. This is fine.